packages/cli/src/commands/source/push/delete.ts

```diff
@@ -104,10 +104,20 @@ export default class SourcePushDelete extends Command {
     const {flags, args} = this.parse(SourcePushDelete);
     const toDelete = `older than ${flags.deleteOlderThan}`;
     try {
-      const res = await source.deleteDocumentsOlderThan(
-        args.sourceId,
-        flags.deleteOlderThan!
-      );
+      const isNumber = flags.deleteOlderThan?.match(/^[0-9]+$/);
```

Comment:
Why not use isNaN

Reply:
Because it is "unreliable".

`new Number('2001/01/01')` -> NaN
`parseInt('2001/01/01')` -> 2000 
`isNaN('2001/01/01)` -> true, but that's not valid TypeScript. `isNan` accept a number, not a string.

I could do `isNaN(new Number(flags.deleteOlderThan))` if you guys prefer/find it more meaningful !

Reply:
No. the regex is simpler. I was just wandering why the regex over the `isNaN`.
Thanks for the explication.

Reply:
```suggestion
      const isNumber = flags.deleteOlderThan?.match(/^\d+$/);
```
I tend to prefer this, but y'know, taste and colors.

The `isNaN(new Number(flags.deleteOlderThan))` is nice too, no RegEx is the best regex often.